### DISCUSSION TITLE
Welcome to ai00_rwkv_server Discussions!

### DISCUSSION THREAD

**Discussion by cgisky1980** [cgisky1980/ai00_rwkv_server#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by digitalw00t** [comment]
Perhaps you could put the discord server invite here for people to join?  I noticed it mentioned in the README, but didn't see an invite.
